src/ngraph/op/fused/mvn.hpp

```diff
@@ -41,18 +41,31 @@ namespace ngraph
                 bool normalize_variance = true,
                 double eps = 1e-9);
 
+            /// \brief Constructs an MVN operation.
+            ///
+            /// \param data Input tensor with data
+            /// \param reduction_axes A list of axes, along which to reduce.
+            /// \param across_channels flag that denotes if mean values are shared across channels.
```

Comment:
I think you meant to delete the `across_channels` line and keep the `normalize_variance` line here.